### DISCUSSION TITLE
If NTP is synchronized, create an event

### DISCUSSION THREAD

**Discussion by DavidTorak** [khoih-prog/ESPAsync_WiFiManager#77] [Q&A]
Hi, can you create an event if NTP time synchronization will take place? 
Something like this: 
**in Setup();**

```
NTP.onNTPSyncEvent([](NTPSyncEvent_t error) {
		if (error) {
			//Serial.print(F("Chyba synchronizace casu: "));
			if (error == noResponse)
				Serial.println(F("NTP server neni dostupny"));
			else if (error == invalidAddress)
				Serial.println(F("Spatna adresa NTP serveru"));
		}
		else {
		
			Serial.print(F("NTP synchronizovano: "));
			}
	});
```

Thanks.

**Comment by khoih-prog** [comment]
Thanks for your interest int the library.

This NTP synchronization has nothing to do with this library, and I won't add this feature. 

Imagine that there are billions of different scenarios and use cases, this WiFiManager library can't and won't be able to provide easy solution for all, but only help in providing a way to help you connect to WiFi and add several important Credentials.

It's extremely easy to use my another [**NTPClient_Generic** library](https://github.com/khoih-prog/NTPClient_Generic) or to write the code yourself to detect if NTP is sync'ed or not

For example, check [**ESP_NTPClient_Advanced.ino**#L96-L99](https://github.com/khoih-prog/NTPClient_Generic/blob/main/examples/ESP/ESP_NTPClient_Advanced/ESP_NTPClient_Advanced.ino#L96-L99)

**Reply by DavidTorak** [reply]
Thank you very much, I totally understand. I've already solved the problem myself. :-)
